public/app/features/migrate-to-cloud/onprem/NameCell.tsx

```diff
@@ -37,6 +37,16 @@ function ResourceInfo({ data }: { data: ResourceTableItem }) {
       return <FolderInfo data={data} />;
     case 'LIBRARY_ELEMENT':
       return <LibraryElementInfo data={data} />;
+    case 'ALERT_RULE':
+      return null;
+    case 'CONTACT_POINT':
+      return null;
+    case 'NOTIFICATION_POLICY':
+      return null;
+    case 'NOTIFICATION_TEMPLATE':
+      return null;
+    case 'MUTE_TIMING':
+      return null;
```

Comment:
FYI - as more resource types come on, and especially if they continue to have generic title/subtitle properties, you could just create a generic ResourceInfo component like 

```tsx
function ResourceInfo({ data }: { data: ResourceTableItem }) {
  switch (data.type) {
    case 'DASHBOARD':
      return <DashboardInfo data={data} />;
    case 'DATASOURCE':
      return <DatasourceInfo data={data} />;
    case 'FOLDER':
      return <FolderInfo data={data} />;
    case 'LIBRARY_ELEMENT':
      return <LibraryElementInfo data={data} />;
    case 'ALERT_RULE':
    case 'CONTACT_POINT':
    case 'NOTIFICATION_POLICY':
    case 'NOTIFICATION_TEMPLATE':
    case 'MUTE_TIMING':
      return <BasicResourceInfo data={data} />;
  }
}

function BasicResourceInfo({ data }: { data: ResourceTableItem }) {
  return (
    <>
      <span>{data.title}</span>
      {data.parentName && <Text color="secondary">{data.parentName}</Text>}
    </>
  );
}
```

or something like that. I don't think each type _requires_ their own component if they're all basically the same.

Reply:
Fair enough! With the work @dana-axinte is doing in #94009 I think we can refactor the FE components to be generic :)